### PR TITLE
day 1 challenge

### PR DESCRIPTION
**Pull request by jimmylorunning**
completed challenge

### CODE REVIEW

File: 2_adventures/001/jimmylorunning/01_day1/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+- `git checkout master`
+- `git remote add upstream https://github.com/RubySteps/21-day-challenge`
+- `git remote -v` (to make sure it's there)
```
Comment (kotp) [inline 2_adventures/001/jimmylorunning/01_day1/README.md:3]: `git remote show` is another way to do this

File: 2_adventures/001/jimmylorunning/01_day1/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- `git pull upstream master` (to get new changes since I last pulled)
+- `git checkout -b 01_day1`
+- `mkdir -p 2_adventures/001/jimmylorunning/01_day1/'`
+- `vim !$README.md`
```
Comment (kotp) [inline 2_adventures/001/jimmylorunning/01_day1/README.md:7]: Very nice use of the !$ last argument variable in bash!  (Just an explanation for those that may not be aware)

File: 2_adventures/001/jimmylorunning/01_day1/README.md (hunk 1 above)
Comment (jimmylorunning) [inline 2_adventures/001/jimmylorunning/01_day1/README.md:3]: cool! I did not know this, thanks :)

File: 2_adventures/001/jimmylorunning/01_day1/README.md (hunk 2 above)
Comment (jimmylorunning) [inline 2_adventures/001/jimmylorunning/01_day1/README.md:7]: Yes! I should've explained. It's a nice trick :). Also !!:x where x is the number of the argument you want... ex:

$ cp hello.txt goodbye.txt
$ vim !!:1  => expands to vim hello.txt
$ vim !!:2  => expands to vim goodbye.txt

File: 2_adventures/001/jimmylorunning/01_day1/README.md (hunk 1 above)
Comment (kotp) [inline 2_adventures/001/jimmylorunning/01_day1/README.md:3]: `git remote show upstream` then to show more details.  Yours is nice for the different information.  An commenting in the requests here is a wonderful way to make notes. :)

**Pull request merged**